<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatBubbleUser</class>
 <widget class="QWidget" name="ChatBubbleUser">
  <layout class="QHBoxLayout" name="bubbleLayout">
   <item>
    <widget class="QLabel" name="labelMessage">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
